src/components/navigation-header/components/primary-nav-submenu/primary-nav-submenu.module.css

```
@@ -0,0 +1,5 @@
+.primarySubnavDropdownItemIcon {
+  --icon-color: var(--product-color, --token-color-foreground-primary);
+
+  color: var(--icon-color);
```

I guess I don't need to define another var here... 
```suggestion
  color: var(--product-color, --token-color-foreground-primary);
```